rootfs/api/models/app.py

```diff
@@ -424,6 +430,7 @@ def _scale_pods(self, scale_types):
                 'app_type': scale_type,
                 'build_type': release.build.type,
                 'healthcheck': healthcheck,
+                'annotations': {'maintenance': app_settings.maintenance},
```

Comment:
This is rather unclear since this is service only but all resource objects in k8s takes annotations